Automattic/wp-calypso
# Desktop App: Update top bar icons and background

Fixes #92824

## Proposed Changes

* Update back, forward, and WP icons to use the latest icons from the WordPress design system.
* Adjusted icon sizing to make them consistent.
* Changed the background color of the top bar to match WordPress.com so it integrates seamlessly.
* Updated icon color and hover state to match WordPress.com.

### Before
<img width="1417" alt="Screenshot 2024-07-19 at 16 07 09" src="https://github.com/user-attachments/assets/554932bb-8503-4a1e-8fdb-a5f2c64190d6">

<img width="1417" alt="Screenshot 2024-07-19 at 16 10 49" src="https://github.com/user-attachments/assets/4588f371-4b86-4447-b04e-1abce21e20ff">


### After
<img width="1417" alt="Screenshot 2024-07-19 at 16 06 48" src="https://github.com/user-attachments/assets/79cbfcba-cac0-46c0-afe6-5f56b975eda7">

<img width="1417" alt="Screenshot 2024-07-19 at 16 10 28" src="https://github.com/user-attachments/assets/8407ed67-8cdf-4bea-81bc-291e1742ebb1">


## Why are these changes being made?


* To ensure consistency across WordPress.com products.

## Testing Instructions



* Apply this PR
* Run the app: `yarn run dev`
* Note that the top bar background and icons have been updated
* Ensure the back, forward, and home actions continue to work

## Pre-merge Checklist
